config/locales/en.yml

```diff
@@ -1912,6 +1912,10 @@ en:
     does_not_match_previous_name: does not match the previous name
   terms_of_service:
     title: Terms of Service
+  terms_of_service_interstitial:
+    preamble_html: We're making some changes to our terms of service, effective <strong>%{date}</strong>. We encourage you to review the updated terms.
```

Comment:
I think this may be wrong. When I sign in and get this message, the terms will long have been in effect already. I think it should be in past tense.

Reply:
I tried to come up with something that would work both for previous and upcoming ToS changes, since someone could log in between the time a ToS change notification was triggered and the time the new ToS apply.

Updated to have a different copy for future and past ToS changes.